docs/pbc/all/business-intelligence/202410.0/amazon-quicksight-third-party-integration/amazon-quicksight.md

```diff
@@ -0,0 +1,118 @@
+
+By looking at datasets, you can see what data from your project is available for adding to analytics. By default, for each dataset, only the data that can be useful for analytics is available from a respective database table. For example, the Customer dataset doesn't include customer passwords.
+
+You can extend the data available to datasets on the project level. For more details, see [Install Amazon QuickSight]().
+
+
+
+### Refreshing data
+
+To make datasets up to date, you need to refresh them by resyncing data from your project or third-party systems.
+
+You can refresh data manually or automatically by setting up a schedule per data set.
+
+
+![refresh-dataset](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/business-intelligence/amazon-quicksight-third-party-integration/amazon-quicksight.md/refresh-dataset.png)
+
+For instructions, see [Refresh analytics datasets](/docs/pbc/all/business-intelligence/202410.0/amazon-quicksight-third-party-integration/back-office-refresh-analytics-datasets.html).
+
+
+
+
+## User management and permissions
+
+
+When working with analytics, users can have the following roles:
+* Author: can view, create, and edit assets.
+* Reader: can view assets.
+
+<!--
+To give a Back Office user access to analytics, you need to assign one of these roles to them. For instructions, see [Create users]() and [Edit users]().
+
+-->
+
+The analytics roles are derived from Amazon QuickSight and are separate from the [Back Office user roles](/docs/pbc/all/user-management/{{page.version}}/base-shop/manage-in-the-back-office/best-practices-manage-users-and-their-permissions-with-roles-and-groups.html).
+
+
+## Analyses and dashboards
+
+In Amazon QuickSight, you are going to work with analyses and dashboards.
+
+An analysis is a customizable visual representation of data. Using the provided tools, you can manipulate your data to extract and present the most important information.
+
+Each analysis can contain multiple data visualizations, which can be rearranged and customized.
+
+
+![QuickSight-analyses](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/business-intelligence/amazon-quicksight-third-party-integration/amazon-quicksight.md/qs-analysis.png)
+
+
+A dashboard is a representation of data prepared in an analysis. It serves as a published version of data analysis, which can be shared with other users.
+
+![QuickSight-dashboard](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/business-intelligence/amazon-quicksight-third-party-integration/amazon-quicksight.md/qs-dashboard.png)
+
+
+Both analyses and dashboards are shareable, you can share a complete dashboard or collaborate on an analysis.
+
+![QuickSight-sharing](https://spryker.s3.eu-central-1.amazonaws.com/docs/pbc/all/business-intelligence/amazon-quicksight-third-party-integration/amazon-quicksight.md/qs-sharing.png)
+
+You can share dashboards with authors and readers, and analyses – only with authors.
+
+### Default dashboards
```

Comment:
let's please add in each of the Sheet sections the show/hide list with the tables with visual and visual type according to this list: https://docs.google.com/spreadsheets/d/1kxFDgNLbHPuWWjm91cwB-MUQSGD44Bh_xqzmHWld69A/edit?gid=1846830203#gid=1846830203 

it will create transparency on OOTB data and avoid answering questions on "what do you track exactly?"

Reply:
what do you mean by sheet sections?